rules/autoconsent/ciaopeople-it.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "ciaopeople.it",
+  "runContext": {
+    "urlPattern": "^https://(www\\.)?(ohga|lexplain|wamily|fanpage)\\.it/"
+  },
+  "prehideSelectors": ["#cp-gdpr-choices"],
+  "detectCmp": [
+    { "waitForVisible": "#cp-gdpr-choices" },
```

Comment:
Let's not delay the `detectCmp` stage, as it will affect the overall performance. We already have the polling logic, so we try multiple times.